ceelogre/incredible-node
# add get functinality for user

#### What does this PR do?
Get user info
#### Description of Task to be completed?
- Create endpoint to get user
- Create user object
#### How should this be manually tested?
- Clone the repo `git clone https://github.com/ceelogre/incredible-node.git`
- Switch your working dirctory `cd incredible-node`
- Checkout the branch `git checkout user-routes-Samuel`
- Install all dependencies `npm install`
- Start the server `npm run dev`
- In your preferred browser hit on `http://localhost:3002/api/users
> You should see user info in JSON format 
#### Any background context you want to provide?
- Sending a get request on the same endpoint using postman should be handy
- Using chrome as the preferred browser is recommended
#### Screenshots (if appropriate)
<img width="800" alt="Screen Shot 2020-10-09 at 08 24 55" src="https://user-images.githubusercontent.com/56896423/95550361-0df3d880-0a09-11eb-94a6-d6af80f383a3.png">
